cli/command/plugin/upgrade.go

```diff
@@ -0,0 +1,41 @@
+package plugin
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/docker/docker/cli"
+	"github.com/docker/docker/cli/command"
+	"github.com/spf13/cobra"
+)
+
+func newUpgradeCommand(dockerCli *command.DockerCli) *cobra.Command {
+	var options pluginOptions
+	cmd := &cobra.Command{
+		Use:   "upgrade [OPTIONS] PLUGIN",
```

Comment:
Upgrade have options similar to install. - disable, --alias string, --disable-content-trust, [KEY=VALUE...]. Better to have a common function for both.